File: Code/java/pattern5.java
import java.util.Scanner;

class Pattern5
{
    public static void main(String[] args)
    {
        int limit, i;
        Scanner scanner = new Scanner(System.in);

        System.out.println("Enter Limit: ");
        limit = scanner.nextInt();

        for (i = limit; i > 0; i--)
            System.out.println("  ".repeat(limit - i) + " *".repeat(i));

        scanner.close();
    }
}

File: Code/cpp/pattern5.cpp
#include<iostream>
using namespace std;
int main()
{
    int limit, i, j;

    cout << "Enter Limit: ";
    cin >> limit;

    for (i = limit; i > 0; i--)
    {
        for (j = 0; j < limit - i; j++)
            cout << "  ";
        for (j = 0; j < i; j++)
            cout << " *";
        cout << "\n";
    }
    return 0;
}